aries-site/src/examples/components/table/TableSingleSelectExample.js

```diff
@@ -193,27 +201,55 @@ const columns = [
   },
 ];
 
-const onClickHandler = record => {
-  // eslint-disable-next-line no-alert
-  alert(`
-    Record was clicked:
-    { 
-        id: ${record.id},
-        orderName: ${record.orderName}
-        state: ${record.state}
-        orderDate: ${record.orderDate}
-    }
-    
-    You can use onClick() to navigate to a record's detail
-    page, open a panel or modal to edit the record, or perform 
-    other actions as you see fit.
-  `);
-};
-
 export const TableSingleSelectExample = () => {
   const size = React.useContext(ResponsiveContext);
+  const [pageDetails, setPageDetals] = React.useState('');
 
-  return (
+  return pageDetails.id !== undefined ? (
+    <>
+      <Button
+        onClick={() => {
+          setPageDetals('');
+        }}
+        alignSelf="start"
+        icon={<FormPrevious />}
+        label="Orders"
+       />
+      <Box margin={{ horizontal: 'large' }} border="bottom">
+        <Heading level={3}>Order Number: {pageDetails.id}</Heading>
+      </Box>
+      <Text
```

Comment:
Thinking this should be an `h2`

Reply:
Whoops I meant "Order Number: OID789345" would be h1 (since it's page title) -- we may adjust size to something small.

Then "Details" should be an h2 (also maybe adjust size to small). double check w designs.